Source/WebInspectorUI/UserInterface/Protocol/InspectorFrontendAPI.js

```diff
@@ -245,4 +245,10 @@ InspectorFrontendAPI = {
     {
         return WI.sharedApp.extensionController.evaluateScriptInExtensionTab(extensionTabID, scriptSource);
     },
+
+    // Returns a string (WI.WebInspectorExtension.ErrorCode) if an error occurred that prevented the resource from loading.
```

Comment:
This comment implies that if the `<iframe>` fails to load (i.e. dispatching an `"error"`) it'd send back the error.  I don't think it's actually doing that.  Should it?  Or should this comment be adjusted?

Reply:
I'll change the comment. The only error returned is if the tab does not exist for that ID.